# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpChannelParent.h
# Commit: be03e0e7b645
# Full Hash: be03e0e7b64568618b834834602b5ba303b32214
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-24 09:31:07
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P3 send OS*R and ODA via pHttpChannel for multipart channel, r=mayhemer,necko-reviewers
#   
#   Depends on D76969
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D76970
# ==============================================================================

diff -r df5abc2c0734 -r be03e0e7b645 netwerk/protocol/http/HttpChannelParent.h
--- a/netwerk/protocol/http/HttpChannelParent.h	Tue Jun 23 16:56:32 2020 +0000
+++ b/netwerk/protocol/http/HttpChannelParent.h	Tue Jun 23 16:56:45 2020 +0000
@@ -356,13 +356,6 @@
   // Used to ensure methods can't be called before OnStartRequest.
   uint8_t mAfterOnStartRequestBegun : 1;
 
-  // Set to true if we get OnStartRequest called with an nsIMultiPartChannel,
-  // and expect multiple OnStartRequest calls.
-  // When this happens we send OnTransportAndData and OnStopRequest over
-  // PHttpChannel instead of PHttpBackgroundChannel to make synchronizing all
-  // the parts easier.
-  uint8_t mIsMultiPart : 1;
-
   // Number of events to wait before actually invoking AsyncOpen on the main
   // channel. For each asynchronous step required before InvokeAsyncOpen, should
   // increase 1 to mAsyncOpenBarrier and invoke TryInvokeAsyncOpen after
